### ISSUE TITLE
MacOS - "Agent was loaded but it is not running" error when running install_service_macos. Running manually works fine

### ISSUE THREAD

**Issue by jpborjas** [Jackett/Jackett#14682]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
Using HTTP Client: HttpWebClient2
Using FlareSolverr: No
Using proxy: Disabled
App config/log directory: /Users/pborjas/.config/Jackett
ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
Running in Docker: No
Jackett variant: CoreMacOs
OS version: Unix 10.13.6 (64bit OS) (64bit process)
Environment version: 6.0.21 (/Users/pborjas/Applications/Jackett/)
Starting Jackett v0.21.756
```

### Description

This is an old installation that has been working fine. After a reboot, I noticed that the application was not starting as a service. I was running a couple of builds behind, but it was running fine. I recreated the plist file using the install_macos_service and saw the error below when I ran the install_service_macos script. Updated to the latest version, same error. 

```text
MacMini2:~ pborjas$ /Users/pborjas/Applications/Jackett/install_service_macos ; exit;
Removing Jackett executable and all .dylib and .dll files from quarantine...
Launching agent...
ERROR: Agent was loaded but is not running. The installation might have failed.
Please open an issue on https://github.com/Jackett/Jackett/issues and paste following information:
*Jackett directory*: /Users/pborjas/Applications/Jackett
*launchctl info*:
org.user.Jackett = {
	active count = 0
	path = /Users/pborjas/Library/LaunchAgents/org.user.Jackett.plist
	state = waiting

	program = /Users/pborjas/Applications/Jackett/jackett
	arguments = {
		/Users/pborjas/Applications/Jackett/jackett
		--NoRestart
	}

	working directory = /Users/pborjas/Applications/Jackett

	inherited environment = {
		Apple_PubSub_Socket_Render => /private/tmp/com.apple.launchd.MMNakdK0pr/Render
		SSH_AUTH_SOCK => /private/tmp/com.apple.launchd.zb06R1yJWs/Listeners
	}

	default environment = {
		PATH => /usr/bin:/bin:/usr/sbin:/sbin
	}

	environment = {
		PATH => /usr/bin:/bin:/usr/sbin:/sbin
		XPC_SERVICE_NAME => org.user.Jackett
	}

	domain = com.apple.xpc.launchd.user.domain.501.100006.Aqua
	asid = 100006
	minimum runtime = 10
	exit timeout = 5
	runs = 0
	successive crashes = 0
	excessive crashing = 0
	last exit code = (never exited)

	event triggers = {
	}

	endpoints = {
	}

	dynamic endpoints = {
	}

	pid-local endpoints = {
	}

	instance-specific endpoints = {
	}

	event channels = {
	}

	sockets = {
	}

	spawn type = daemon
	spawn role = (null)
	jetsam priority = 3
	jetsam memory limit (active) = (unlimited)
	jetsam memory limit (inactive) = (unlimited)
	jetsamproperties category = daemon
	cpumon = default

	properties = {
		partial import = 0
		launchd bundle = 0
		xpc bundle = 0
		keepalive = 1
		runatload = 1
		dirty at shutdown = 0
		low priority i/o = 0
		low priority background i/o = 0
		legacy timer behavior = 0
		exception handler = 0
		multiple instances = 0
		supports transactions = 0
		supports pressured exit = 0
		enter kdp before kill = 0
		wait for debugger = 0
		app = 0
		system app = 0
		creates session = 0
		inetd-compatible = 0
		inetd listener = 0
		abandon process group = 0
		one-shot = 0
		event monitor = 0
		penalty box = 0
		pended non-demand spawn = 1
		role account = 0
		launch only once = 0
		system support = 0
		app-like = 0
		inferred program = 1
		joins gui session = 0
		joins host session = 0
		parameterized sandbox = 0
		resolve program = 0
		abandon coalition = 0
		high bits aslr = 0
		extension = 0
		nano allocator = 0
		no initgroups = 0
		start on fs mount = 0
		endpoints initialized = 1
		disallow all lookups = 0
		system service = 0
	}
}
*LaunchAgents permissions*:
drwxr-xr-x   9 pborjas  staff   288 Sep  8 12:56 .
drwx------@ 70 pborjas  staff  2240 Mar 28 13:23 ..
-rw-r--r--   1 pborjas  staff   712 Sep  8 12:56 org.user.Jackett.plist

logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]
```

### Logged Error Messages

```text
ERROR: Agent was loaded but is not running. The installation might have failed.
```

**Comment by ilike2burnthing** [comment]
Retry multiple times, as per https://github.com/Jackett/Jackett/issues/12572.

Failing that, uninstall and delete any leftover Jackett files and directories (you may wish to backup `~/.config/Jackett/` first). If not being used by anything else, delete `~/.config/` as well. Then restart MacOS and try reinstalling Jackett with freshly downloaded files. Again, you may need to retry multiple times.
